docs/specification.md

```diff
@@ -746,14 +702,11 @@ Parameter (in Michelson)
   (address %sender)
   (pair
     (address %minter)
-    (pair
-      (nat %tokenId)
-      (nat %amount)
-)))
+    (nat %amount)
+))
 ```
 
-- Set the amount of allowed minting allowance for an address with some
-  specific token type.
+- Set the amount of allowed minting allowance for an address.
 
 - Minter must present in sender's minter list.
```

Comment:
But not necessarly in the global minter list?

Reply:
I don't touch it in the first part yet. Currently I don't understand the situation with minter lists. I will update it in the next part.